packages/core-p2p/src/peer.ts

```diff
@@ -31,6 +31,7 @@ export class Peer implements P2P.IPeer {
     public state: any;
     public url: string;
     public lastPinged: dayjs.Dayjs | null;
+    public verification?: PeerVerificationResult;
```

Comment:
What does the `?` designate? It is not described in https://www.typescriptlang.org/docs/handbook/classes.html

Reply:
Removed the `?` and added `| null`, similar to `lastPinged` in 245502691